proto/podium/api/v1/podium.proto

```diff
@@ -0,0 +1,371 @@
+      get: "/l/{leaderboard_id}/members/{member_public_id}"
+    };
+  }
+
+  // GetMembers retrieves information about multiple members of a leaderboard.
+  rpc GetMembers(GetMembersRequest) returns (GetMembersResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/members"
+    };
+  }
+
+  // RemoveMember removes a member from a leaderboard.
+  rpc RemoveMember(RemoveMemberRequest) returns (BasicResponse) {
+    option (google.api.http) = {
+      delete: "/l/{leaderboard_id}/members/{member_public_id}"
+    };
+  }
+
+  // RemoveMembers allow the removal of multiple members of a leaderboard.
+  rpc RemoveMembers(RemoveMembersRequest) returns (BasicResponse) {
+    option (google.api.http) = {
+      delete: "/l/{leaderboard_id}/members"
+    };
+  }
+
+  // GetRank retrieves ranking information about a member.
+  rpc GetRank(GetRankRequest) returns (GetRankResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/members/{member_public_id}/rank"
+    };
+  }
+
+  // GetAroundMember retrieves the closest members to another member on the leaderboard.
+  rpc GetAroundMember(GetAroundMemberRequest) returns (MemberListResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/members/{member_public_id}/around"
+    };
+  }
+
+  // GetAroundScore retrieves the closest members to a score on the leaderboard.
+  rpc GetAroundScore(GetAroundScoreRequest) returns (MemberListResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/scores/{score}/around"
+    };
+  }
+
+  // GetTopMembers retrieves the top ranking members of a leaderboard.
+  rpc GetTopMembers(GetTopMembersRequest) returns (MemberListResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/top/{page_number}"
+    };
+  }
+
+  // GetTopPercentage retrieves a percentage of the top members of the leaderboard.
+  rpc GetTopPercentage(GetTopPercentageRequest) returns (MemberListResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/top-percent/{percentage}"
+    };
+  }
+
+  // UpsertScoreMultiLeaderboards sends a member score to multiple leaderboards.
+  rpc UpsertScoreMultiLeaderboards(MultiUpsertScoreRequest) returns (MultiUpsertScoreResponse) {
+    option (google.api.http) = {
+      put: "/m/{member_public_id}/scores"
+      body: "score_multi_change"
+    };
+  }
+
+  // GetRankMultiLeaderboards retrieves information about a member in multiple leaderboards.
+  rpc GetRankMultiLeaderboards(MultiGetRankRequest) returns (MultiGetRankResponse) {
+    option (google.api.http) = {
+      get: "/m/{member_public_id}/scores"
+    };
+  }
+}
+
+message HealthCheckResponse {
+  string working_string = 1;
+}
+
+message StatusResponse {
+  double error_rate = 1;
+}
+
+message RemoveLeaderboardRequest {
+  string leaderboard_id = 1;
+}
+
+// ScoreUpsert allow to provide score information about a single member.
+message ScoreUpsert {
+  string publicID = 1;
```

Comment:
nit: public_id.

Same throughout.

Reply:
Need to maintain publicID because of how this is transformed into a struct. If I use a public_id ... it will be transformed into a publicId and will break backward compatibility.

Reply:
Give that proto tag we talked about a try and let me know how it goes.

`string public_id = 1 [json_name="publicID"];`

Reply:
Checked out [json_name] on .proto. Seems promising but there was a strange behaviour for collections (the names didn't exported correctly for them). Will look into this on a later moment.